### PR TITLE
residual multichannel->channel_axis fixes

### PR DESCRIPTION
**Pull request by grlee77**
## Description

The first commit here cleans up a few residual uses of `multichannel` in docstring examples, parameter descriptions and warning texts.

The second commit is to update the benchmark scripts with a `_channel_kwarg` helper that will specify `multichannel` when benchmarking against `skimage<0.19`, but `channel_axis` on newer versions.

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

**Review by mkcor — APPROVED** [review]
Yay! Thank you, @grlee77 :pray:

File: skimage/transform/_warps.py [hunk 1]
```diff
@@ -1124,14 +1123,12 @@ def warp_polar(image, center=None, *, radius=None, output_shape=None,
     """
     multichannel = channel_axis is not None
     if image.ndim != 2 and not multichannel:
-        raise ValueError("Input array must be 2 dimensions "
-                         "when `multichannel=False`,"
-                         " got {}".format(image.ndim))
+        raise ValueError("Input array must be 2 dimensional when "
```
Comment (mkcor) [inline skimage/transform/_warps.py:1126]:
```suggestion
        raise ValueError("Input array must be 2-dimensional when "
```

File: skimage/transform/_warps.py [hunk 2]
```diff
@@ -1124,14 +1123,12 @@ def warp_polar(image, center=None, *, radius=None, output_shape=None,
...
+                         f"`channel_axis=None`, got {image.ndim}")
 
     if image.ndim != 3 and multichannel:
-        raise ValueError("Input array must be 3 dimensions "
-                         "when `multichannel=True`,"
-                         " got {}".format(image.ndim))
+        raise ValueError("Input array must be 3 dimensional when "
```
Comment (mkcor) [inline skimage/transform/_warps.py:1130]:
```suggestion
        raise ValueError("Input array must be 3-dimensional when "
```

**Comment by rfezzani** [comment]
:tada: Thank you @grlee77 for your hard work and @mkcor for the reminder :wink:

**Pull request merged**